### PR TITLE
Add nbde cockpit lab

### PR DESCRIPTION
**Pull request by sabbaka**
I still need to add description about accessing Cockpit and some actual screenshots for Storage page.

### CODE REVIEW

**Comment by sabbaka** [comment]
Please, review and if everything looks good feel free to merge.

File: 2019Labs/RHELSecurityLab/documentation/lab3_NBDE_Cockpit.adoc [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+Your NBDE2 server is already encrypted and this exercise will install the client packages that will allow you to un-encrypt the root device drive upon boot without manual intervention.
+
+. Navigate to the Lab information page from *Lab 0 Setup steps*. This page has your environment's power control and consoles. Click on the console for *NBDE2*. The passphrase for the LUKs encrypted disk on *NBDE2* is *r3dh4t1!*. You will need to enter the pass phrase to complete the boot process.  If you wanted to copy and paste text to send to the console, press the *T* button at the top right of your console.
++
+image:images/lab3-console2.png[200,200]
+image:images/console-textbox.png[500,500]
+
+. Login to the NBDE2 server as *root* using the password *r3dh4t1!*.
+. Then, install the clevis packages:
++
```
Comment (dkopecek) [inline 2019Labs/RHELSecurityLab/documentation/lab3_NBDE_Cockpit.adoc]: This mentions clevis but the command contains cockpit-storaged

File: 2019Labs/RHELSecurityLab/documentation/lab3_NBDE_Cockpit.adoc [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+[source, text]
+[root@nbde2 ~]# yum install cockpit-storaged
+. Next, we will initialize the luks binding to the tang server. If you wanted to copy and paste the text below to send to the console, press the *T* button at the top right of your console (See the picture above on Step #1).
++
+
+== Lab 3.2 Installing the cockpit-storaged
+[root@nbde3 ~] # yum install cockpit-storaged
```
Comment (dkopecek) [inline 2019Labs/RHELSecurityLab/documentation/lab3_NBDE_Cockpit.adoc]: This part doesn't render correctly (github view of the doc).

File: 2019Labs/RHELSecurityLab/documentation/lab3_NBDE_Cockpit.adoc [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+Your NBDE2 server is already encrypted and this exercise will install the client packages that will allow you to un-encrypt the root device drive upon boot without manual intervention.
+
+. Navigate to the Lab information page from *Lab 0 Setup steps*. This page has your environment's power control and consoles. Click on the console for *NBDE2*. The passphrase for the LUKs encrypted disk on *NBDE2* is *r3dh4t1!*. You will need to enter the pass phrase to complete the boot process.  If you wanted to copy and paste text to send to the console, press the *T* button at the top right of your console.
++
+image:images/lab3-console2.png[200,200]
+image:images/console-textbox.png[500,500]
+
+. Login to the NBDE2 server as *root* using the password *r3dh4t1!*.
+. Then, install the cockpit-session-recording package:
++
```
Comment (dkopecek) [inline 2019Labs/RHELSecurityLab/documentation/lab3_NBDE_Cockpit.adoc]: Again, mentions cockpit-session-recording but installs storaged. Looks like a c&p error :-)

File: 2019Labs/RHELSecurityLab/documentation/lab3_NBDE_Cockpit.adoc (hunk 1 above)
Comment (sabbaka) [reply 2019Labs/RHELSecurityLab/documentation/lab3_NBDE_Cockpit.adoc]: Fixed.

File: 2019Labs/RHELSecurityLab/documentation/lab3_NBDE_Cockpit.adoc (hunk 2 above)
Comment (sabbaka) [reply 2019Labs/RHELSecurityLab/documentation/lab3_NBDE_Cockpit.adoc]: Fixed.

File: 2019Labs/RHELSecurityLab/documentation/lab3_NBDE_Cockpit.adoc (hunk 3 above)
Comment (sabbaka) [reply 2019Labs/RHELSecurityLab/documentation/lab3_NBDE_Cockpit.adoc]: Fixed.

**Comment by sabbaka** [comment]
Lab is updated.

**Pull request merged**